biztalk/core/how-to-configure-the-xml-disassembler-pipeline-component.md

```diff
@@ -33,6 +33,7 @@ The XML Disassembler pipeline component should be used in the Disassemble stage
     |--------------|----------------|  
     |**Allow unrecognized message**|Indicates whether to allow messages that do not have a recognized message type to be passed through the disassembler.<br /><br /> Default value: **False**|  
     |**Document schemas**|Indicates the namespace and typename of the schema or schemas to be applied to the document. For more information, see [How to Use the Schema Collection Property Editor](../core/how-to-use-the-schema-collection-property-editor.md).<br /><br /> Schemas specified in this property should have unique target namespaces. If any of the schemas have the same namespace, the validation of the document instances may not work as expected. If schemas must have the same namespace, you should either create a separate pipeline for each schema and specify one schema per XML Disassembler pipeline component or use one pipeline but do not specify any schemas as parameters for the XML Disassembler pipeline component.<br /><br /> Default value: Empty collection|  
+    |**DtdProcessing**|Indicates whether to allow DTD processing within the pipeline. This property is available **BizTalk Server 2020** onwards. <br /> When **Parse** or empty, DTD processing will be enabled.<br /> When **Ignore**, DOCTYPE element in the incoming XML messages will be ignored and no DTD processing will occur.<br /> When **Prohibit"**, DTD processing will be disabled and any incoming XML messages that use DTD will get suspended.<br /><br /> Default value: empty|  
```

Comment:
Extra quote " in prohibit, not sure if its intentional.

Reply:
thanks! good catch! 👍